CHANGELOG.md

```diff
@@ -4,11 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## Unreleased
+### Changed
+- Nothing should go in this section, please add to the latest unreleased version, or
```

Comment:
The release automation doesn't make commits back to git, so the changelog needs to be in a releasable state at all times (and version numbers are derived from it).  I would see the date on the version number be the last time it was updated, so if you add an entry, you bump the date up until that version is released.  I'm completely open to wording things differently here.  parse-a-changelog requires an `Unreleased` section, so the thinking was we needed an indicator of some sort to remind people it's no longer the right place.

Reply:
Roger that. Maybe it would help to have a reminder to bump the date, e.g. "... please add to the latest unreleased version (and update the corresponding date) ..."?